cookbook/plugins/direct-database-actions/work-with-quantity/readme.md

```diff
@@ -0,0 +1,42 @@
+# plugin-name: "work-with-quantity"
+
+
+# General Introduction
+This plugin creates one flow intended to serve as example for handling Quantity and Unit utilizing SLIMSGate api. 
+
+## Usage: _Content Management_
+## Type: *slimsgate*
+
+# Flow
+
+<details>
+
+<summary>
+
+### <a name="Quantity"></a> Work with Quantity
+
+</summary>
+
+#### User inputs:
+None
+#### Process
+- Fetch a content record with the configured barcode in the property _contentBarcode_
+- Retrieve the volume (field 'cntn_cf_volume'), this is a Map<String, Object>, and convert/cast it to a Quantity Object using the service *QuantityService*
+- Create a new Quantity Object with the Unit configured in the property _quantityToSubtractUnitName_ using the service *UnitService* and a hard-coded amount of 2 
+- Compare the two Quantity Objects to make sure that the volume of this content is bigger or equal to the Quantity created, because we want to reduce the volume of the content with this Quantity and want to avoid having a negative quantity
+- Finally, converting the reduced volume Quantity to another configured unit in the property _finalUnitName_ using the service *UnitService* and using this value to update the hard-coded field 'cntn_cf_reducedVolume'
```

Comment:
For things like 'cntn_cf_reducedVolume' I tend to prefer using ` characters as they have the "no-format" text look and it helps things stand out more.

So rather than 'cntn_cf_reducedVolume'  `cntn_cf_reducedVolume` -- but feel free to differ in preference.